### PR TITLE
De-scope the Data Integrity specification to focus on Verifiable Credentials.

### PR DESCRIPTION
**Pull request by msporny**
This PR de-scopes the Data Integrity specification to primarily focus on Verifiable Credentials.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/32.html" title="Last updated on Aug 15, 2022, 3:17 PM UTC (a8110f0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/32/3d32fae...a8110f0.html" title="Last updated on Aug 15, 2022, 3:17 PM UTC (a8110f0)">Diff</a>

### CODE REVIEW

**Review by dlongley — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Comment by peacekeeper** [comment]
To me it seems a bit strange to give something that's generically applicable (not just to VCs) such a restrictive name. This feels like renaming generic JWT to VC-JWT, even though JWT can be used for other non-VC applications.

But I won't object if others support the change.

**Comment by msporny** [comment]
> To me it seems a bit strange to give something that's generically applicable (not just to VCs) such a restrictive name. This feels like renaming generic JWT to VC-JWT, even though JWT can be used for other non-VC applications.

I agree, I'd rather keep the specification name aligned with what it's capable of doing. The danger here is that people end up thinking that Data Integrity is only useful when used with Verifiable Credentials, when it is designed to be a generalized data integrity mechanism. While I'm not in favor of this change, I won't object to it as long as actions are not taken to establish DI as a VC-only thing. If we see that occurring, then we will want to undo this PR and re-align long-term expectations.

**Comment by msporny** [comment]
> > To me it seems a bit strange to give something that's generically applicable (not just to VCs) such a restrictive name. This feels like renaming generic JWT to VC-JWT, even though JWT can be used for other non-VC applications.

I agree, I'd rather keep the specification name aligned with what it's capable of doing. The danger here is that people end up thinking that Data Integrity is only useful when used with Verifiable Credentials, when it is designed to be a generalized data integrity mechanism. While I'm not in favor of this change, I won't object to it as long as actions are not taken to establish DI as a VC-only thing. If we see that occurring, then we will want to undo this PR and re-align long-term expectations.

**Comment by mprorock** [comment]
I think If we define it well for VCs we can always look at generalizing later.  If we try to generalize early (even though we can obviously apply some of the techniques here broadly) we risk not getting the core matter of Data Integrity as related to VCs tightly and well defined.  We need extremely well defined and thoroughly picked apart and inspected data integrity for VCs and we should focus first and foremost on ensuring that is in place.  Then if we want to look at other cases of how to use this with broader signing of JSON-LD and similar data cool, but let's get this well proven and testable in the VC only use case first.

File: index.html [hunk 1]
```diff
@@ -138,8 +141,9 @@
     <section id='abstract'>
       <p>
 This specification describes mechanisms for ensuring the authenticity and
-integrity of structured digital documents using cryptography, such as digital
-signatures and other digital mathematical proofs.
+integrity of Verifiable Credentials and other types of structured digital
+documents using cryptography, such as digital signatures and other digital
+mathematical proofs.
```
Comment (mprorock) [inline index.html]:
```suggestion
integrity of Verifiable Credentials using cryptography and especially through the use of digital signatures and related mathematical proofs.
```

**Review by mprorock — APPROVED** [review]
minor suggestion.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: -1, the removal of "and other types of structured digital documents using cryptography" is exactly the sort of goal post shifting that is of concern with this PR. The following modification to your proposal could work:

```suggestion
integrity of Verifiable Credentials and similar types of constrained digital
documents using cryptography, especially through the use of digital 
signatures and related mathematical proofs.
```

Comment (mprorock) [reply index.html]: Good with this

**Review by OR13 — APPROVED** [review]

**Review by vongohren — APPROVED** [review]
Based on discussions it looks like a good plan

**Comment by msporny** [comment]
Substantive, multiple reviews, no strong objections (but cautious approval), merging.

**Pull request merged**